bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/SonosXMLParser.java

```diff
@@ -761,25 +744,25 @@ public void startElement(@Nullable String uri, @Nullable String localName, @Null
                 case "res":
                     currentElement = CurrentElement.res;
                     break;
-                case "streamContent":
+                case "r:streamContent":
```

Comment:
Same in this method, are you 100% sure of all the "prefix" you added ?

Reply:
Yes.